buku.py

```diff
@@ -2543,8 +2557,5 @@ def main():
     if args.fixtags:
         bdb.fixtags()
 
-    # Close DB connection and quit
```

Comment:
I was getting a 'cannot operate on a closed cursor error', and after I took that line out the error stopped.